### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by suda**
Includes docs for #121 so can be merged after it.

### CODE REVIEW

**Comment by jerone** [comment]
Travis CI is failing because MarkDown is linted also: https://github.com/suda/tool-bar/pull/116

**Comment by suda** [comment]
Ok, makes sense :) Fixed linter warnings

File: README.md [hunk 1]
```diff
@@ -146,22 +155,33 @@ consumeToolBar: (toolBar) ->
     # Teardown any stateful code that depends on tool bar ...
 ```
 
+## Methods
+
+### `.addButton({icon, callback, priority, tooltip, data})`
```
Comment (jerone) [inline README.md]: So helpful!

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
 		"url": "https://github.com/suda/tool-bar"
 	},
 	"scripts": {
-		"lint": "node_modules\\.bin\\coffeelint . && node_modules\\.bin\\remark ."
+		"lint": "coffeelint . && remark ."
```
Comment (jerone) [inline package.json]: :+1:

**Pull request merged**